index.html

```diff
@@ -353,14 +353,22 @@ <h1>Important Terms</h1>
 			<dd>
 			  <p>Usable by users in ways they can control. References in this document relate to <cite><a href="https://www.w3.org/TR/WCAG21/#operable"><abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.1 Principle 2: Content must be operable</a></cite> [[WCAG21]]. See <a>Keyboard Accessible</a>.</p>
 			</dd>
-			<dt><dfn data-export="" data-dfn-for="ARIA" data-local-lt="owned|owned element's">Owned Element</dfn></dt>
+			<dt><dfn data-export="" data-lt="accessibility child|owned child|child element|child|children|child elements">Accessibility child</dfn></dt>
+			<dd>
+				<p>An 'accessibility child' is a <abbr title="Document Object Model">DOM</abbr> child of the <a>element</a>, a descendant of the <a>element</a> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening, a child specified via an <pref>aria-owns</pref> relationship to the element, or a descendant of an element with role <rref>generic</rref> or <rref>none</rref> specified via <pref>aria-owns</pref> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening.</p>
```

Comment:
i wonder if this could be written as a list of conditions?  i'm finding this incredibly difficult to read.

maybe if examples were added?  i had to write out an example for each to make sure i was actually following along.

DOM child
```
<div role=listbox>
  <div role=option ...>...</div>
</div>
```

Descendent of an element
```
<div role=listbox>
  <div>
    <div role=option ...>...</div>
  </div>
</div>
```

aria-owns relationship.
```
<div role=listbox aria-owns=f></div>
<div role=option id=f ...>...</div>
```

aria-owns relationship with intervening elements.
this one definitely seems covered.
```
<div role=listbox aria-owns=f></div>
<div id=f>
  <div>
    <div role=option ...>...</div>
  </div>
</div>
```

this one seems like it has to be covered, but the wording keeps making me think maybe it could not be?
```
<div role=listbox aria-owns=f></div>
<div id=f>
  <div role=option ...>...</div>
</div>
```

i'm a bit stuck on the 'with only...' where that to me says what follows is part of the condition.  But 'with or without only' to me makes both of my above examples true.

sorry, i wish i hadn't spent the last 35 minutes just on this one line.

Reply:
```suggestion
				<p>An 'accessibility child' refers to the child in the <a>accessibility tree<a> ignoring any intervening elements of role <rref>generic</rref> or <rref>none</rref>.</p>
				<p>Related: <a href="#mustContain">Allowed Accessibility Child Roles</a>, <a>Accessibility Parent</a>, <pref>aria-owns</pref><p>
```

Ok, I decided to try my hand at this, what do you guys think with a similar definition for  accessibility parent? I think it covers all of @scottaohara's and @smhigley cases. It was @pkra review of my definition of hidden which makes me propose this (see hidden: https://w3c.github.io/aria/#dfn-hidden)

Reply:
Hm, I actually think this goes too far in the direction of simplification. I think I'd prefer to explicitly list the ways the child can be associated with the parent, i.e.
- DOM child
- DOM descendent with only generics intervening
- direct aria-owns relationship
- descendant of aria-owns relationship with only generics intervening

I think the issue with "child in the accessibility tree" is it becomes a sort of circular definition -- an "accessibility child" is a "child in the accessibility tree", but without specifying how to construct the accessibility tree.

Reply:
I'll add the list of examples though, with one for each :)